homeassistant/components/nibe_heatpump/climate.py

```diff
@@ -0,0 +1,349 @@
+"""The Nibe Heat Pump climate."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from nibe.coil import Coil
+from nibe.exceptions import CoilNotFoundException
+
+from homeassistant.components.climate import (
+    ATTR_HVAC_MODE,
+    ATTR_TARGET_TEMP_HIGH,
+    ATTR_TARGET_TEMP_LOW,
+    ATTR_TEMPERATURE,
+    ClimateEntity,
+    ClimateEntityDescription,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import DOMAIN, LOGGER, Coordinator
+from .const import Series
+
+
+@dataclass
+class ClimateDescriptionMixin:
```

Comment:
```suggestion
class NibeClimateDescriptionMixin:
```

The name got me when reviewing this since it took a minute to realize it was being defined by the integration